add "Sink Flush Rows OPS" panel for tidb cloud metrics integration panel (#2775)
DataDog/integrations-extras: tidb_cloud/CHANGELOG.md

# CHANGELOG - TiDB Cloud

## 1.2.0 / 2025-08-22

***Added***:

> * Add a panel "Sink Flush Rows OPS" for “TiDB Cloud Dynamic Tracker” dashboard.

Reviewer: ```suggestion
* Added "Sink Flush Rows OPS" widget to the TiDB Cloud Dynamic Tracker dashboard.
```
